source/package-json.d.ts

```diff
@@ -257,8 +257,26 @@ declare namespace PackageJson {
 		typings?: string;
 	}
 
+	export interface Workspaces {
+		/**
+		The packages blob to use for workspaces.
+		*/
+		packages: string[];
+		/**
+		A list of glob patterns for packages that should not be hoisted.
```

Comment:
I would elaborate on what "hoisting" mean here.

Reply:
Done, plus I've added a link to the Yarn website.